poi-ooxml/src/main/java/org/apache/poi/ooxml/HyperlinkRelationship.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.poi.ooxml;

```

Comment:
can you add an Apache source header here?

Reply:
Done